src/components/WarningAcknowledgement/WarningAcknowledgement.tsx

```diff
@@ -0,0 +1,184 @@
+import { Box, Button } from '@chakra-ui/react'
+import type { AnimationDefinition, MotionStyle } from 'framer-motion'
+import { AnimatePresence, motion } from 'framer-motion'
+import type { PropsWithChildren } from 'react'
+import React, { useCallback, useEffect, useState } from 'react'
+import { FiAlertTriangle } from 'react-icons/fi'
+import { useTranslate } from 'react-polyglot'
+import { RawText, Text } from 'components/Text'
+
+const initialProps = { opacity: 0 }
+const animateProps = { opacity: 1 }
+const exitProps = { opacity: 0, transition: { duration: 0.5 } }
+const transitionProps = { delay: 0.2, duration: 0.1 }
+
+const WarningOverlay: React.FC<PropsWithChildren> = ({ children }) => {
+  return (
+    <motion.div
+      key='overlay'
+      style={{
+        backgroundColor: 'var(--chakra-colors-blanket)',
+        position: 'absolute',
+        top: 0,
+        bottom: 0,
+        left: 0,
+        right: 0,
+        zIndex: 4,
+      }}
```

Comment:
this can be popped out into a const

Reply:
Popped in https://github.com/shapeshift/web/pull/6647/commits/a80e3298a7be7cadeb5b5b00768e1b219ada3cf6